src/uct/ib/rdmacm/rdmacm_ep.c

```diff
@@ -236,12 +236,12 @@ void uct_rdmacm_ep_set_failed(uct_iface_t *iface, uct_ep_h ep, ucs_status_t stat
     uct_rdmacm_iface_t *rdmacm_iface = ucs_derived_of(iface, uct_rdmacm_iface_t);
     uct_rdmacm_ep_t *rdmacm_ep       = ucs_derived_of(ep, uct_rdmacm_ep_t);
 
+    rdmacm_ep->status = status;
```

Comment:
failing tests: 
*/test_ucp_sockaddr.err_handle/2, 
 rdmacm_ep.c:200  UCX  WARN  destroying endpoint 0x437b910 with not completed operation

Reply:
@evgeny-leksikov

Reply:
if I understand correctly, because before this change rdmacm_ep was destroyed immediately from set_ep_failed, now - from API call, so this is just bugfix according to new err handling behavior